### ISSUE TITLE
another unlabeled button

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#56]
**Description**  
Here is another unlabeled button with screen readers

**How to Reproduce**  
1. Go to settings > accessibility > talkback and turn it on
2. Open thunder > open sidebar > open a community
3. Tap the icon highlighted in the screenshot below

**Expected Behavior**  
The button should be labeled

**Screenshots**  
![Screenshot_20230624-133011_thunder](https://github.com/hjiangsu/thunder/assets/116814522/2bcd5be4-73a4-4ee8-82dc-dc48094f3bcd)

**Device & App Version:**  
 - Device: [e.g. iPhone6] Motorola One 5G Ace
 - OS: [e.g. iOS8.1] Android 11
 - Version [e.g. 22] prerelease found here https://github.com/hjiangsu/thunder/releases/tag/v0.2.1-prerelease%2B5

**Additional Context**  
It is getting really hard to find these. Great work

**Comment by hjiangsu** [comment]
I have added this in the latest release v0.2.1+6. I'll close this issue but let me know if it looks okay!
